astro_2.0/features/CreateProposal/helpers/rootHelper.tsx

```diff
@@ -804,6 +817,13 @@ export function getValidationSchema(
           ),
         details: yup.string().required('Required'),
         externalUrl: yup.string().url(),
+        gas: yup
+          .number()
+          .typeError('Must be a valid number.')
+          .positive()
+          .min(0.01)
+          .max(0.25)
+          .required('Required'),
```

Comment:
Maybe, there is sense to place this validation in a variable and reuse it everywhere?